policies/templates/gcp_kubernetes_legacy_abac_v1.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Check to see if GKE Legacy ABAC is disabled.
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-kubernetes-legacy-abac-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GKELegacyAbacConstraintV1
+        plural: GKELegacyAbacConstraintsV1
+      validation:
+        openAPIV3Schema:
+          properties: {}
+  targets:
+    validation.gcp.forsetisecurity.org:
+      rego: |
+        #INLINE("validator/gke_legacy_abac.rego")
```

Comment:
Please run `make build` (and `make format`) to put latest code in template.

Reply:
Updated